src/components/NewLexemeForm.vue

```diff
@@ -28,6 +28,16 @@ const lexicalCategory = computed( {
 		store.commit( SET_LEXICAL_CATEGORY, newLexicalCategory );
 	},
 } );
+
+const submitMsg = $messages.get( 'wikibaselexeme-newlexeme-submit' );
+const termsOfUseURL = 'https://foundation.wikimedia.org/wiki/Terms_of_Use';
+const copyrightText = $messages.get(
+	'wikibase-shortcopyrightwarning',
+	submitMsg,
+	termsOfUseURL,
+	store.state.config.licenseUrl,
+	store.state.config.licenseName,
+);
```

Comment:
This doesn’t seem to match the version of the `wikibase-shortcopyrightwarning` message that I see in `repo/i18n/en.json` – that message only has three parameters, not four, and the second one looks like it should be a title instead of a full URL.